lib/common/util.js

```diff
@@ -124,7 +124,7 @@ util.arrayize = arrayize;
  */
 function format(template, args) {
   return template.replace(/{([^}]*)}/g, function(match, key) {
-    return args[key] || match;
+    return key in args ? args[key] : match;
```

Comment:
It previously used the value of `args[key]` to decide if it should be used in the resulting string. Instead, we should confirm the presence of the key in the provided object and use its value, regardless of if it passes a truthy test.

This wasn't an issue until in this PR where we needed something similar to:

```js
format('/api/path{pathExtension}', {
  pathExtension: condition ? 'extension' : ''
});
```

Previous result if `condition` was false: `/api/path{pathExtension}`
New result: `/api/path`